### DISCUSSION TITLE
Support `referrerPolicy` in `<Avatar />`

### DISCUSSION THREAD

**Discussion by sethmbrown** [mantinedev/mantine#5116] [Feature requests]
When using [GoogleUserContent](https://lh3.googleusercontent.com/), the prop `referrerPolicy="no-referrer"` is required to have the avatar fetched without throwing a `403`. Only update needed for Mantine's `<Avatar />` is to add `referrerPolicy` to the type definition. Despite the type error currently, it seems the behavior works as expected with the current `<Avatar />` implementation. Requesting only a type definite change to allow `referrerPolicy`.

`referrerPolicy` docs - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy
Related stackoverflow - https://stackoverflow.com/questions/56242788/http-403-on-images-loaded-from-googleusercontent-com

Thanks!

**Comment by rtivital** [comment]
You can pass props down to the `img` element with `imageProps` – https://codesandbox.io/s/relaxed-golick-vk496r?file=/src/App.tsx

**Reply by sethmbrown** [reply]
Thank you, @rtivital!
